src/routes/api/installation/api-installation-delete-pollinator.ts

```diff
@@ -0,0 +1,58 @@
+import { Request, Response } from "express";
+import { Subscription } from "models/subscription";
+import { getJiraClient } from "~/src/jira/client/jira-client";
+import { RepoSyncState } from "~/src/models/reposyncstate";
+
+export const STAGE_POLLINATOR_JIRA_HOST = "https://fusion-arc-pollinator-staging-app.atlassian.net";
+export const PROD_POLLINATOR_JIRA_HOST = "https://fusion-pollinator.atlassian.net";
+
+export const ApiInstallationDeletePollinator = async (req: Request, res: Response): Promise<void> => {
+	const gitHubInstallationId = req.params.installationId;
+	const gitHubAppId = Number(req.params.gitHubAppId) || undefined;
+	const jiraHost = req.params.jiraHost;
+
+	if (!jiraHost || !gitHubInstallationId) {
+		const msg = "Missing Jira Host or Installation ID";
+		req.log.warn({ req, res }, msg);
+		res.status(400).send(msg);
+		return;
+	}
+
+	if (jiraHost !== STAGE_POLLINATOR_JIRA_HOST && jiraHost !== PROD_POLLINATOR_JIRA_HOST) {
+		const msg = "Jira Host not a pollinator jira site";
+		req.log.warn({ req, res }, msg);
+		res.status(400).send(msg);
+		return;
+	}
+
+	const subscription: Subscription | null = await Subscription.getSingleInstallation(
+		jiraHost,
+		Number(gitHubInstallationId),
```

Comment:
Can you `Number(gitHubInstallationId)` up here `const gitHubInstallationId = req.params.installationId;`
should still be safe for `if (!jiraHost || !gitHubInstallationId)`?

Reply:
good call :) done